### PR TITLE
Update harmonized system data for 2022 version

### PR DESCRIPTION
**Pull request by hlmartin**
Hello! This pull request contains the updated dataset for HS 2022.

The canonical source is still the [WCO site](http://www.wcoomd.org/en/topics/nomenclature/instrument-and-tools/hs-nomenclature-2022-edition.aspx). However, I was able to find the 2022 version available as JSON on the Comtrade data extraction API. It's not directly linked on the documentation page, but I noticed I could iterate from HS 2012 to HS 2022 by changing the URL:

(HS 2012) https://comtrade.un.org/Data/cache/classificationH4.json
(HS 2022) https://comtrade.un.org/Data/cache/classificationH6.json

😅 

I've left in my (rather basic) Python conversion script of this classification data to the format used in this dataset, but happy to have that removed if you don't see a use in keeping it around.

Please let me know if there's anything else you'd like from me around this PR. Cheers!

### CODE REVIEW

File: convert.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  elif chapter <= 92:
+    return "XVIII"
+  elif chapter == 93:
+    return "XIX"
+  elif chapter <= 96:
+    return "XX"
+  elif chapter == 97:
+    return "XXI"
+  else:
+    return "TOTAL"
```
Comment (boronia) [inline convert.py:50]: just curious... does `TOTAL` means "no parent"?

**Review by boronia — APPROVED** [review]
neat

**Comment by sabas** [comment]
@hlmartin Hi, thank you very much for this!!!

Could you please move the extraction script under a scripts folder like in this other repo
https://github.com/datasets/un-locode

**Review by sabas — CHANGES_REQUESTED** [review]
Move script to subfolder per https://specs.frictionlessdata.io/data-package/#illustrative-structure

**Comment by hlmartin** [comment]
Thanks @sabas! I've moved the script and added a little bit of user friendliness (readme + argument parsing) while I was at it.

File: datapackage.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
   ],
   "sources": [
     {
-      "name": "UN Comtrade Reference Tables",
-      "path": "https://unstats.un.org/unsd/tradekb/Knowledgebase/50039/UN-Comtrade-Reference-Tables",
+      "name": "UN Comtrade data extraction API",
+      "path": "https://comtrade.un.org/data/doc/api/",
```
Comment (mikelkew) [inline datapackage.json:18]: Should this link be updated in the main [README.md](https://github.com/datasets/harmonized-system/blob/e739a58b8dad763bc4b58958f89fa90897db0c35/README.md) as well?

Comment (sabas) [reply datapackage.json:18]: Yes it should. After this change we can merge it @hlmartin

Comment (hlmartin) [reply datapackage.json:18]: Fixed.

**Comment by sabas** [comment]
🎉

**Review by saeedshkoore — APPROVED** [review]

**Pull request merged**